src/js/events/MetronomeClient.ts

```diff
@@ -142,7 +142,7 @@ const defaultHeaders = {
   "Content-Type": "application/json; charset=utf-8"
 };
 
-export function createJob(data: JobData) {
+export function createJob(data: JobDetailResponse): Observable<any> {
```

Comment:
Can we please type the response as we need it right now? I postponed writing the types until we need them, but now we do need them, so it would be cool to have types here.